sparse/numba_backend/__init__.py

```diff
@@ -157,10 +165,16 @@
     where,
 )
 from ._dok import DOK
+from ._info import capabilities, default_device, default_dtypes, devices, dtypes
```

Comment:
I think we shouldn't import them to the main namespace. Let's keep them in `__array_namespace_info__` only.

Reply:
The current array API tests require these in the main namespace as well.

Reply:
Hmm then I think it's a bug in the `array-api-tests` suite. FYI in NumPy Aaron suggested we should wait a bit with bumping Array API to 2023 until `array-api-tests` is ready: https://github.com/numpy/numpy/pull/26572#issuecomment-2140832356

Reply:
Array API standard doesn't say they need to be in the main namespace: https://data-apis.org/array-api/latest/API_specification/inspection.html#inspection-apis

Reply:
Demonstrated the failure in https://github.com/pydata/sparse/pull/696/commits/42c9bfd1e7bfbed42550ceec3d803d6a53b3497b, let's wait for `array-api-tests` fixes as you suggest.